scoring/templates/scoring/base.html

```diff
@@ -69,8 +69,8 @@
         </div>
     </nav>
     <nav class="subheader bg-blue-gradient">
-        <span>Please help Climate Emergency UK by supporting our Crowdfunder. We rely on donations to remain independent.</span>
-        <a class="btn donation ml-2" href="https://www.crowdfunder.co.uk/p/council-climate-scorecards">Donate now</a>
+        <span>Please help support Climate Emergency UK. We rely on donations to remain independent.</span>
+        <form action="https://www.paypal.com/donate" method="post" target="_top"><input type="hidden" name="hosted_button_id" value="JB823YSUV5X9S"><input id="ceuk-donate" type="image" src="//www.paypalobjects.com/en_GB/i/btn/btn_donate_LG.gif" name="submit" title="PayPal - The safer, easier way to pay online!" alt="Donate with PayPal button" border="0"><img alt="" src="//www.paypal.com/en_GB/i/scr/pixel.gif" width="1" height="1" border="0"></form>
```

Comment:
Such a small thing, but I _think_ the `alt` text of the `<input type="image">` here is what will get read out in assistive devices (it is, at least, in VoiceOver on Mac) and as a result, users will hear something like:

> "Donate with PayPal button"

or worse (as is the case in VoiceOver):

> "Donate with PayPal button, button"

I’d drop the "button" from the `alt` text – it makes sense without it.

Reply:
No worries @zarino I'll take a look on this sprint.

Reply:
I've pushed a fixup for both these things.